crates/storage/src/schema/revision_0055.rs

```diff
@@ -4,18 +4,20 @@ pub(crate) fn migrate(tx: &rusqlite::Transaction<'_>) -> anyhow::Result<()> {
     tx.execute_batch(
         r"
         CREATE TABLE trie_class_removals (
-            block_number INTEGER PRIMARY KEY,
+            block_number INTEGER NOT NULL,
             indices BLOB
         );
+        CREATE INDEX trie_class_removals_block_number ON trie_class_removals(block_number);
         CREATE TABLE trie_contracts_removals (
             block_number INTEGER NOT NULL,
             indices BLOB
         );
         CREATE INDEX trie_contracts_removals_block_number ON trie_contracts_removals(block_number);
         CREATE TABLE trie_storage_removals (
-            block_number INTEGER PRIMARY KEY,
+            block_number INTEGER NOT NULL,
```

Comment:
iiuc this is required so that we don't have to merge the existing values with the new nodes being coalesced?

Reply:
Yes, exactly.